lib/index.js

```diff
@@ -50,7 +50,7 @@ const gcIsSafe = {
   So: true // Symbol, Other
 }
 
-export default async version => {
+export default async (version, options) => {
```

Comment:
```suggestion
export default async (version, { safeCategories = gcIsSafe } = {}) => {
```

And I think we can rename `gcIsSafe` to `DEFAULT_SAFE_CATEGORIES`.